### PR TITLE
hot topics design unification

### PR DESCRIPTION
**Pull request by andygnewman**
screen shot 2015-08-26 at 11 46 27
screen shot 2015-08-26 at 11 46 43

### CODE REVIEW

File: client/components/stream/_hot-topics.scss [hunk 1]
```diff
@@ -73,30 +68,21 @@
...
-	position: absolute;
-	top: 0;
-	right: 0;
-	bottom: 0;
-	left: 0;
-	padding: 10px;
-	color: white;
+	@include nTypeBravo(4);
+	white-space: nowrap;
+	width: calc(100% -16px);
```
Comment (ironsidevsquincy) [inline client/components/stream/_hot-topics.scss]: how does this look in IE<9?

File: client/components/stream/_hot-topics.scss [hunk 2]
```diff
@@ -73,30 +68,21 @@
...
+	overflow: hidden;
+	text-overflow: ellipsis;
+	text-transform: uppercase;
+	text-align: center;
+	padding: 8px;
+	color: oColorsGetPaletteColor('white');
 
 	// IE8-9 don't display the image
-	background-color: oColorsGetPaletteColor('red-1');
+	background-color: oColorsGetPaletteColor('claret');
```
Comment (adgad) [inline client/components/stream/_hot-topics.scss]: could this be `oColorsGetColorFor(topic, text)`

Comment (ironsidevsquincy) [inline client/components/stream/_hot-topics.scss]: and there's also an alias for `oColorsGetPaletteColor`- https://github.com/Financial-Times/next-sass-setup/blob/master/_functions.scss#L5

Comment (adgad) [inline client/components/stream/_hot-topics.scss]: Yes, I can add that to next-sass-setup now

Comment (andygnewman) [inline client/components/stream/_hot-topics.scss]: @adgad 
As it's a claret background in this instance calling getColorFor(topic, text) would be confusing.

Comment (adgad) [inline client/components/stream/_hot-topics.scss]: Hmmm...maybe we could have a topic-block use case? with background claret and text=white...

File: client/components/stream/_hot-topics.scss (hunk 1 above)
Comment (andygnewman) [inline client/components/stream/_hot-topics.scss]: @ironsidevsquincy 
in IE9 - no image
screen shot 2015-08-26 at 15 36 05

this is how it currently looks on live in IE9
screen shot 2015-08-26 at 15 40 46

Comment (ironsidevsquincy) [inline client/components/stream/_hot-topics.scss]: what about 8? I more just wondering, as `calc` is unavailable for <9, what's the consequences visually?

Comment (andygnewman) [inline client/components/stream/_hot-topics.scss]: Managed to get my IE8 VM working, looked OK, but I did revisit the need for calc and found I didn't need it, so good challenge.

File: client/components/stream/_hot-topics.scss [hunk 3]
```diff
@@ -73,30 +68,20 @@
...
-	right: 0;
-	bottom: 0;
-	left: 0;
-	padding: 10px;
-	color: white;
+	@include nTypeBravo(4);
+	white-space: nowrap;
+	overflow: hidden;
+	text-overflow: ellipsis;
+	text-transform: uppercase;
```
Comment (i-like-robots) [inline client/components/stream/_hot-topics.scss]: We get in trouble every time we do this so beware!

File: client/components/stream/_hot-topics.scss [hunk 4]
```diff
@@ -114,28 +99,27 @@
...
+	@include nTypeCharlie(4);
+	@include nLinksHeadline('article-comment');
+	margin: 0;
 }
 
 .hot-topics__list-item-date {
-	color: oColorsGetPaletteColor('cold-1');
+	@include nTypeAlpha(5);
+	text-transform: uppercase;
+	color: getColorFor(timestamp, text);
```
Comment (i-like-robots) [inline client/components/stream/_hot-topics.scss]: other strings are quoted so let's keep it consistent

File: client/components/stream/_hot-topics.scss (hunk 3 above)
Comment (andygnewman) [inline client/components/stream/_hot-topics.scss]: Ok, will await the wrath of editorial, but they get the win of not having the title over the image.

File: client/components/stream/_hot-topics.scss (hunk 4 above)
Comment (andygnewman) [inline client/components/stream/_hot-topics.scss]: OK will change it here (and in the one from the original implementation of hot topics that I copied the syntax from)

**Pull request merged**